sqlite-android/build.gradle

```diff
@@ -160,15 +169,15 @@ publishing {
             }
         }
     }
-    repositories {
-        maven {
-            url "https://oss.sonatype.org/service/local/staging/deploy/maven2"
-            credentials {
-                username properties.getProperty('sonatype.username')
-                password properties.getProperty('sonatype.password')
-            }
-        }
-    }
```

Comment:
jitpack doesn't have a properties file with those values so it can't handle this